### PR TITLE
[E2ETest][UWP]Adding protocol activation to UWP E2ETest

### PR DESCRIPTION
**Pull request by gb92**
By adding <uap:Protocol Name="zumoe2e-uwp"> the UWP app can now be launched using the powershell command: start-process zumoe2e-uwp: with the configuration settings passed in as query string parameters. 

Also fixed error with PhoneIdentity change

### CODE REVIEW

File: e2etest/UWP.E2ETest/App.xaml.cs [hunk 1]
```diff
@@ -97,6 +98,39 @@ protected override void OnLaunched(LaunchActivatedEventArgs args)
             Window.Current.Activate();
         }
 
+        protected override void OnActivated(IActivatedEventArgs args)
+        {
+            
+            if(args.Kind == ActivationKind.Protocol)
```
Comment (dhei) [inline e2etest/UWP.E2ETest/App.xaml.cs]: nitpick: space needed in "if(".

File: e2etest/UWP.E2ETest/App.xaml.cs [hunk 2]
```diff
@@ -10,6 +10,7 @@
 using Windows.UI.Xaml.Controls;
 using Windows.UI.Xaml.Navigation;
 using Microsoft.WindowsAzure.MobileServices.TestFramework;
+using System.Net.Http;
```
Comment (dhei) [inline e2etest/UWP.E2ETest/App.xaml.cs]: nitpick: re-ordering the using statements

File: e2etest/UWP.E2ETest/App.xaml.cs [hunk 3]
```diff
@@ -97,6 +98,39 @@ protected override void OnLaunched(LaunchActivatedEventArgs args)
...
 
+        protected override void OnActivated(IActivatedEventArgs args)
+        {
+            
+            if(args.Kind == ActivationKind.Protocol)
+            {
+                ProtocolActivatedEventArgs protocolArgs = args as ProtocolActivatedEventArgs;
+
+                Frame content = Window.Current.Content as Frame;
+                if(content == null)
```
Comment (dhei) [inline e2etest/UWP.E2ETest/App.xaml.cs]: nitpick: space needed in "if(".

File: e2etest/E2ETestFramework/Helper/TestConfigHelper.cs [hunk 4]
```diff
@@ -90,5 +117,34 @@ private static string getArgumentValue(string name, Dictionary<string, string> a
...
+        /// </summary>
+        /// <param name="query">String containing the query string</param>
+        /// <returns>Dictionary representing the arguments collection</returns>
+        private static Dictionary<string, string> parseQueryString(string query)
+        {
+            if(query.IndexOf("?") == 0)
+            {
+                query = query.Substring(1);
+            }
+
```
Comment (dhei) [inline e2etest/E2ETestFramework/Helper/TestConfigHelper.cs]: Can you do more parameter validation of `string query`? Such as check if `query` is null or empty, check if `query` starts with "?". It's good even for private method only used internally.

File: e2etest/UWP.E2ETest/App.xaml.cs (hunk 2 above)
Comment (gb92) [inline e2etest/UWP.E2ETest/App.xaml.cs]: using was added and then unused. removed

File: e2etest/E2ETestFramework/Helper/TestConfigHelper.cs (hunk 4 above)
Comment (gb92) [inline e2etest/E2ETestFramework/Helper/TestConfigHelper.cs]: Added a check that will throw an argument exception if the argument is null or empty

**Comment by dhei** [comment]
Looks good :+1:

**Pull request merged**